src/main/java/seedu/toluist/ui/MainWindow.java

```diff
@@ -126,10 +127,20 @@ private void configureUndoKeyCombination() {
         FxViewUtil.setKeyCombination(getRoot(), keyCombination, handler);
     }
 
+    private void configureHistoryNavigationKeyPresses() {
+        String[] keyNames = new String[] { "up", "down"};
+        Arrays.stream(keyNames).forEach(keyName -> {
+            KeyCode keycode = getKeyCode(keyName);
+            String navigateCommand = "navigatehistory " + keyName;
+            EventHandler<ActionEvent> handler = event -> dispatcher.dispatch(navigateCommand);
+            FxViewUtil.setKeyCode(getRoot(), keycode, handler);
```

Comment:
thats provided you dont want to provide user with the ability to use up/down to navigate the task list later on